seatunnel-core/seatunnel-starter/src/main/java/org/apache/seatunnel/core/starter/seatunnel/args/ClientCommandArgs.java

```diff
@@ -65,6 +69,10 @@ public class ClientCommandArgs extends AbstractCommandArgs {
         description = "list job status")
     private boolean listJob = false;
 
+    @Parameter(names = {"-cj", "--close-job"},
+        description = "Close client the task will also be closed")
+    private boolean closeJob = false;
```

Comment:
```suggestion
    private boolean closeJob = true;
```

Reply:
Default is true? Since the client is blocking, maybe the user wants the task to continue?

Reply:
Just my opinon, I think user use shut down command to cancel job is intuitive operation. If they want to continue running task after client closed, they can use `-cj`